noxfile.py

```diff
@@ -0,0 +1,201 @@
+"""Nox sessions."""
+import os
+import sys
+
+import nox
+from nox_poetry import Session, session
+
+python_versions = ["3.11"]
+nox.options.envdir = ".cache"
+# To run consecutive nox sessions faster.
+nox.options.reuse_existing_virtualenvs = False
+nox.options.sessions = (
+    "lint",
+    "mypy",
+    "openapi",
+    "safety",
+    "unit_tests",
+    # "integration_tests",
+    # "contract_tests",
+)
+
+
+@session(python=python_versions[0])
+def tests(session: Session) -> None:
+    """Run all tests."""
+    session.notify("unit_tests")
+    # session.notify("integration_tests")
+    # session.notify("contract_tests")
+
+
+@session(python=python_versions[0])
+def openapi(session: Session) -> None:
+    """Generate API spec from code."""
+    session.install(".")
+    session.install("PyYAML")
+    session.run("python", "openapi.py")
```

Comment:
Denne mangler openapi.py
```suggestion
@session(python=python_versions[0])
def openapi(session: Session) -> None:
    """Generate API spec from code."""
    session.install(".")
    session.install("PyYAML")
    session.run("python", "openapi.py")
```